src/clinicalstudy.ts

```diff
@@ -141,7 +141,7 @@ export type EnrollmentStruct =
 // Arm Group
 
 export interface ArmGroupStruct {
-  arm_group_label: One<string>;
+  arm_group_label?: One<string>;
```

Comment:
According to the schema `arm_group_label` is not optional - is it missing in some of the results returned?

Reply:
No, you are correct. I have now removed this in d3a48a3